source/_posts/2015-06-03-technical-thursdays-or-how-we-do-continuous-learning.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Technical Thursdays or: how we do continuous learning
+hero_image: /blog/images/heros/post-high.jpg
+comments: true
+author:
+  name: Sebastián Martínez
+  email: sebastian@wyeworks.com
+  twitter_handle: smartinez87
+  github_handle:  smartinez87
+  image:  /images/team/sebastian-martinez.jpg
+  description: VP of Engineering at WyeWorks. Ruby on Rails hacker. ExceptionNotification maintainer. Coffee & bacon lover.
+published: true
+---
+
+A conclusion I came up with after years of watching software projects come and go, is that one of the biggest differences between teams who achieve consistent success and those who suffer through inconsistency is their commitment to continuous learning.
```

Comment:
Cambiaría _A conclusion I came up with after years ..._ por algo como _A conclusion I came to after years..._.
_come up with_ me suena más a inventarse algo, y _come to_ como llegar a algo.